<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDeviceDialog</class>
 <widget class="QDialog" name="CreateDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stw√≥rz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adres IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>25</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Czujniki:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>81</y>
     <width>19</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>3:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>4:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>81</y>
     <width>74</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="comboBox0">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox1">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox2">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox3">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
